assembly-image.gemspec

```diff
@@ -1,11 +1,10 @@
 # frozen_string_literal: true
 
 $LOAD_PATH.push File.expand_path('lib', __dir__)
-require 'assembly-image/version'
 
 Gem::Specification.new do |s|
   s.name        = 'assembly-image'
-  s.version     = Assembly::Image::VERSION
+  s.version     = '1.8.0'
```

Comment:
because `Assembly::Image` is now a subclass of a class in the `assembly-objectfile` gem, we can't have the VERSION file in the appropriate Assembly::Image namespace AND have the version value be part of the gemspec.